### PR TITLE
Add find in parent directory functionality

### PR DESCRIPTION
**Pull request by codingbelief**
This request is to add finding the selected word in the parent directory functionality and mapping to 'alt-a' key.
Finding in parent directory is frequently used when reading souce code.
And using hot key is more convenient than using the 'Search In Directory' context menu command.

### CODE REVIEW

File: keymaps/find-and-replace.cson [hunk 1]
```diff
@@ -3,6 +3,9 @@
   'cmd-f': 'find-and-replace:show'
   'cmd-alt-f': 'find-and-replace:show-replace'
 
+'atom-text-editor':
+  'alt-a': 'project-find:show-in-parent-directory'
```
Comment (benogle) [inline keymaps/find-and-replace.cson]: Why `alt-a` Is it used in other editors?

**Comment by benogle** [comment]
This seems like the same functionality as the `show-in-current-directory`. Can you make the codepaths the same and use `show-in-current-directory` for the keybinding?

**Comment by benogle** [comment]
Also there is a linter error, and this does not have specs.

File: keymaps/find-and-replace.cson (hunk 1 above)
Comment (codingbelief) [inline keymaps/find-and-replace.cson]: I'm not finding any other editors using `alt-a`, I just can't find other free keybinding of two keys which can be handled by one hand. 

I try to use `alt-f` at first, but it is used as 'editor:move-to-end-of-word' by atom core.

**Comment by timkelty** [comment]
This works for me, but I have to do the keymap twice - once brings up find (empty), a second populates the current dir.

**Comment by timkelty** [comment]
@codingbelief I believe having to do the keymap twice is a result of having the following setting:
```
  tabs:
    usePreviewTabs: true
```

If the current file is fully open (you have double clicked it from tree-view or edited it), then `show-in-current-directory` works right away. However, if you are previewing the file, you have to do `show-in-current-directory` twice to get the path in there.

**Comment by timkelty** [comment]
Is there anything that can be done to move this along?
I've been running @codingbelief's fork for a couple weeks now.

The only issue I've noticed is the compatibility with the `usePreviewTabs` option, as mentioned above.

**Comment by timkelty** [comment]
@codingbelief @benogle anything I can do to move this along?

**Comment by benogle** [comment]
This needs a spec. 

cc @lee-dohm

**Comment by timkelty** [comment]
@codingbelief if you merge https://github.com/codingbelief/find-and-replace/pull/2, the CI test should pass.

**Comment by codingbelief** [comment]
Sorry for reply late.

**Comment by codingbelief** [comment]
I'm not familiar with spec, @timkelty  can you help to deal with the new error?

**Comment by timkelty** [comment]
I'm not really either :(. I can give it a shot if I get time, though.

**Comment by timkelty** [comment]
This seems pretty simple, but I don't think many on this thread are familiar enough to make a spec.

Is anyone able to jump in and help? /cc @lee-dohm @benogle

**Comment by lee-dohm** [comment]
I'm buried right now ... but let me see if I can find someone to chip in.

**Comment by timkelty** [comment]
Thanks!

**Comment by travco** [comment]
@codingbelief Pending merge in codingbelief#3 to resolve current CI failing check, and add two tests to spec for the functionality added.

**Comment by codingbelief** [comment]
@travco  Thanks for you help. ～

**Comment by timkelty** [comment]
@lee-dohm Can we get this merged in now that it has some tests?

**Comment by lee-dohm** [comment]
Looks good to me. Thanks for contributing!

**Pull request merged**